main.go

```diff
@@ -97,18 +98,6 @@ func (s Step) run() {
 	stepconf.Print(cfgs)
 	fmt.Println()
 
-	// Project-or-Workspace flag
-	action := ""
-	if strings.HasSuffix(cfgs.ProjectPath, ".xcodeproj") {
-		action = "-project"
-	} else if strings.HasSuffix(cfgs.ProjectPath, ".xcworkspace") {
-		action = "-workspace"
-	} else {
-		failf("Invalid project file (%s), extension should be (.xcodeproj/.xcworkspace)", cfgs.ProjectPath)
-	}
-
-	log.Printf("* action: %s", action)
-
```

Comment:
I have removed this for a couple of reasons:
- The iOS version of this step does not have such an explicit check
- It was not clear how useful this is because we log this selection as `* action:` but the whether the command uses a project or workspace does not really count as an action
- The xcodebuild command will immediately fail with an error message if something else is supplied than xcodeproj, xcworkspace or Package.swift file

Based on these I did not find this useful.